main.go

```diff
@@ -1,5 +1,28 @@
 package main
 
+import (
+	"flag"
+	"fmt"
+
+	"../inventory/inventory/view"
```

Comment:
make go mod module as `github.com/sankethkini/InventoryInGo`
this import will become `"github.com/sankethkini/InventoryInGo/inventory/view"`